CLI/table.c

```diff
@@ -874,19 +889,23 @@ int do_table_get(int argc, char **argv)
         goto clean_up;
 
     /* 3. Get key */
-    if (parse_table_key(&argc, &argv, &entry) != NO_ERROR)
-        goto clean_up;
+    bool key_provided = (argc >= 1 && is_keyword(*argv, "key"));
+    if (key_provided) {
+        if (parse_table_key(&argc, &argv, &entry) != NO_ERROR)
+            goto clean_up;
+    }
 
     if (argc > 0) {
         fprintf(stderr, "%s: unused argument\n", *argv);
         goto clean_up;
     }
 
-    error_code = psabpf_table_entry_get(&ctx, &entry);
-    if (error_code != NO_ERROR)
-        goto clean_up;
-
-    error_code = print_json_table_entry(&ctx, &entry, table_name);
+    if (key_provided) {
+        error_code = psabpf_table_entry_get(&ctx, &entry);
+        if (error_code != NO_ERROR)
+            goto clean_up;
+    }
+    error_code = print_json_table_entry(&ctx, key_provided ? &entry : NULL, table_name);
```

Comment:
Using `else` avoids second check `key_provided ? &entry`

Reply:
With `else` funtion `print_json_table_entry` must be called twice, one in `if` and one in `else` block. Now I think that better name for this function will be `print_json_table`. `entry` is provided only when single entry must be printed.